docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -148,6 +148,29 @@ oc get secret nova-migration-ssh-key || oc create secret generic nova-migration-
 rm -f id*
 cd -
 ----
++
+
+. Set LIBVIRT_PASSWORD to match the existing {OpenStackShort} deployment password:
++
+----
+declare -A TRIPLEO_PASSWORDS
+TRIPLEO_PASSWORDS[default]="$HOME/overcloud-passwords.yaml"
+LIBVIRT_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' LibvirtTLSPassword:' | awk -F ': ' '{ print $2; }')
+----
+* Create libvirt-secret when TLS-e is enabled.
+[source,yaml]
+----
+$ oc apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: libvirt-secret
+  namespace: openstack
+type: Opaque
+data:
+  LibvirtPassword: ${LIBVIRT_PASSWORD}
```

Comment:
This needs to be base64 encoded. Will link an internal thread on Slack.

Reply:
Did you test the code you are proposing -- does it result in the correct password being written to the compute nodes config?

All available info indicates that if we provide just `LibvirtPassword: ${LIBVIRT_PASSWORD}`, OpenShift will interpret that value as base64 whether we like it or not. So it will try to run base64 decoding operation on that, and then i'd expect it to either fail, or if the password happens to be a valid base64 by chance, it would get mangled into something incorrect.

Please see the conversation i linked on Slack.

Reply:
This also help to see how the secrets work: https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html-single/deploying_red_hat_openstack_services_on_openshift/index#proc_creating-the-data-plane-secrets_dataplane

Reply:
There is a confusion between `oc set data` and plain `oc apply -f yaml`. You are correct if we apply yaml we need to base64 the password. I've updated PR. Thank you